editor/renames_map_3_to_4.cpp

```diff
@@ -570,26 +581,21 @@ const char *RenamesMap3To4::gdscript_function_renames[][2] = {
 	{ "viewport_set_use_arvr", "viewport_set_use_xr" }, // RenderingServer
 	{ "warp_mouse_position", "warp_mouse" }, // Input
 	{ "world_to_map", "local_to_map" }, // TileMap, GridMap
-	{ "set_shader_param", "set_shader_parameter" }, // ShaderMaterial
-	{ "get_shader_param", "get_shader_parameter" }, // ShaderMaterial
-	{ "set_uniform_name", "set_parameter_name" }, // ParameterRef
-	{ "get_uniform_name", "get_parameter_name" }, // ParameterRef
 
 	// Builtin types
 	// Remember to add them to the builtin_types_excluded_functions variable, because for now these functions cannot be listed.
-	//{ "empty", "is_empty" }, // Array -- Used as custom rule. Be careful, this will be used everywhere.
-	//{ "remove", "remove_at" }, // Array -- Breaks Directory and several more.
+	// { "empty", "is_empty" }, // Array -- Used as custom rule. Be careful, this will be used everywhere.
+	// { "invert", "reverse" }, // Array -- Give it a check. Be careful, this will be used everywhere.
+	// { "remove", "remove_at" }, // Array -- Breaks Directory and several more.
 	{ "clamped", "limit_length" }, // Vector2
 	{ "get_rotation_quat", "get_rotation_quaternion" }, // Basis
 	{ "grow_margin", "grow_side" }, // Rect2
-	{ "invert", "reverse" }, // Array -- Give it a check. Be careful, this will be used everywhere.
 	{ "is_abs_path", "is_absolute_path" }, // String
 	{ "is_valid_integer", "is_valid_int" }, // String
 	{ "linear_interpolate", "lerp" }, // Color
 	{ "find_last", "rfind" }, // Array, String
 	{ "to_ascii", "to_ascii_buffer" }, // String
 	{ "to_utf8", "to_utf8_buffer" }, // String
-	{ "to_wchar", "to_utf32_buffer" }, // String -- utf32 or utf16?
```

Comment:
Wrong (will be fixed by https://github.com/godotengine/godot/pull/73225).

Reply:
So, shouldn't it be changed to convert to `to_wchar_buffer` then?

Reply:
It's not merged yet, because production folks are stringy on their merges :P

Reply:
Ahh, right right, the vermin of this world get in our way again!